From: Bernd Zeimetz <bernd@bzed.de>
Date: Thu, 22 Apr 2010 20:52:00 +0200
Subject: [PATCH] MAKE is needed when setup.py is running the clean command.

---
 setup.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/setup.py b/setup.py
index a6d2bad..6795557 100644
--- a/setup.py
+++ b/setup.py
@@ -37,6 +37,7 @@ except ValueError:
 if not manpages:
     print("No XML processor, omitting manual-page installation.")
 
+MAKE = ("MAKE" in os.environ) and os.environ["MAKE"] or "make"
 if not 'clean' in sys.argv:
     abs_builddir = ("abs_builddir" in os.environ) and os.environ["abs_builddir"] or ""
     if not os.path.exists(os.path.join(abs_builddir, 'gpsd_config.h')):
@@ -44,7 +45,6 @@ if not 'clean' in sys.argv:
         sys.exit(1)
 
     cdcmd = abs_builddir and ("cd '" + abs_builddir + "' && ") or ""
-    MAKE = ("MAKE" in os.environ) and os.environ["MAKE"] or "make"
     for f_name in needed_files:
         # TODO:  Shouldn't make be run unconditionally in case a
         # dependency of f_name has been updated?
-- 
