From 18328a21fd975738df2240e186e836a0c1cda528 Mon Sep 17 00:00:00 2001
From: Bernd Zeimetz <bernd@bzed.de>
Date: Thu, 15 Jul 2010 00:39:35 +0200
Subject: [PATCH 1/2] Create pkg_config and prl files for Qgpsmm.

---
 libQgpsmm/libQgpsmm.pro |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/libQgpsmm/libQgpsmm.pro b/libQgpsmm/libQgpsmm.pro
index 13a33fd..7faed4c 100644
--- a/libQgpsmm/libQgpsmm.pro
+++ b/libQgpsmm/libQgpsmm.pro
@@ -77,6 +77,12 @@ HEADERS += libQgpsmm_global.h \
 
     QMAKE_CFLAGS += -D_GNU_SOURCE
 
+    # create pkg_config and prl files
+    CONFIG += create_pc create_prl
+    QMAKE_PKGCONFIG_REQUIRES = QtNetwork
+    QMAKE_PKGCONFIG_DESTDIR = pkgconfig
+    pkgconfig.path = $${TARGET_LIBDIR}
+    INSTALLS += pkgconfig
 }
 
 win32 {
-- 
1.7.1

