From: Bernd Zeimetz <bernd@bzed.de>
Date: Fri, 23 Apr 2010 02:59:31 +0200
Subject: [PATCH] Make make silen during version detection in setup.py.

---
 setup.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/setup.py b/setup.py
index 6795557..d8b512a 100644
--- a/setup.py
+++ b/setup.py
@@ -60,7 +60,7 @@ if not 'clean' in sys.argv:
 gpspacket_sources = ["gpspacket.c", "packet.c", "isgps.c",
             "driver_rtcm2.c", "strl.c", "hex.c", "crc24q.c"]
 include_dirs = [ os.path.realpath(os.path.dirname(__file__)) ]
-version_out = os.popen(MAKE + " version")
+version_out = os.popen(MAKE + " -s version")
 version = version_out.read()
 print(version)
 if version_out.close():
-- 
