From 17d27339a809343a6f5c5e0fe2e020bdf19f1f6a Mon Sep 17 00:00:00 2001
From: Chris Kuethe <ckuethe@users.berlios.de>
Date: Thu, 15 Jul 2010 14:13:02 -0700
Subject: [PATCH] fix the way skyview json is produced

---
 gpsd_json.c |    6 ++----
 1 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/gpsd_json.c b/gpsd_json.c
index 42ead71..45456bf 100644
--- a/gpsd_json.c
+++ b/gpsd_json.c
@@ -261,15 +261,13 @@ void json_sky_dump(const struct gps_data_t *datap,
 			       datap->ss[i], used ? "true" : "false");
 	    }
 	}
-    }
-    if (reported) {
 	if (reply[strlen(reply) - 1] == ',')
 	    reply[strlen(reply) - 1] = '\0';	/* trim trailing comma */
-	(void)strlcat(reply, "]", replylen - strlen(reply));
+	(void)strlcat(reply, "]", replylen);
     }
     if (reply[strlen(reply) - 1] == ',')
 	reply[strlen(reply) - 1] = '\0';	/* trim trailing comma */
-    (void)strlcat(reply, "}\r\n", replylen - strlen(reply));
+    (void)strlcat(reply, "}\r\n", replylen);
     if (datap->satellites_visible != reported)
 	gpsd_report(LOG_WARN, "Satellite count %d != PRN count %d\n",
 		    datap->satellites_visible, reported);
-- 
1.7.1

